app/styles/main.scss

```diff
@@ -76,12 +76,10 @@ p {
   margin-right: 10px;
 }
 
-.ng-enter {
+.ng-enter.enter {
```

Comment:
I find `enter` rather confusing, especially because it is combined with the reserved `.ng-enter`. This class name should be more clear.